package.json

```diff
@@ -75,6 +78,8 @@
     "chai-datetime": "1.5.0",
     "coveralls": "3.0.2",
     "cross-env": "3.1.3",
+    "db-migrate": "0.11.5",
+    "db-migrate-mongodb": "false1.5.0",
```

Comment:
I think we decided you were going to do whatever you think is best with this.

Reply:
I removed the dummy migration